readthedocs/projects/forms.py

```diff
@@ -706,6 +706,11 @@ def clean_domain(self):
             domain_string = parsed.netloc
         else:
             domain_string = parsed.path
+
+        if Domain.objects.filter(domain=domain_string).exists():
+            raise forms.ValidationError(
+                _('This domain is not available.')
+            )
```

Comment:
This is causing an error when re-saving an existing domain. It should check for that case. 

<img width="525" alt="screenshot 2019-01-29 17 12 52" src="https://user-images.githubusercontent.com/25510/51944082-1f3db280-23e9-11e9-83b4-546586cc45ae.png">

Reply:
What do you actually mean by this?
I have added a `if` for this condition and another suitable error message.